index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Joseph Kwesi Kwentsir Portfolio</title>
+</head>
+<body>
+    <header class="page-header">
+      <nav class="nav-header"> 
+          <ul>
+              <li>Welcome</li>
+              <li> <a href="#"> <img src="/images/Union.svg" alt=""/></a></li>
+          </ul>
+      </nav>
+    </header>
+    
+
+      <section class="page-headline">
+        <h2 class="intro">
+            Hey there. I’m Joseph
+            <br>
+            <span class="job">I’m a software developer</span>
+        </h2> 
+        <p class="intro-work">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.
+        </p>
+        <ul class="social-handle-icons">
+            <li class="twitter">
+                <a href="#"> <img src="/images/twitter.svg" alt="Twitter Icon"></a> 
+            </li>
+            <li class="linkedin">
+                <a href="#"> <img src="/images/Linkedin.svg" alt="Linkedin Icon"></a>
+            </li>
+            <li class="medium">
+                <a href="#"> <img src="/images/medium.svg" alt="Medium Icon"></a>
+            </li>
+            <li class="github">
+                <a href="#"> <img src="/images/github.svg" alt="Github Icon"></a>
+            </li>
+            <li class="angelist">
+                <a href="#"> <img src="/images/angellist.svg" alt="Angelist Icon"></a>
+            </li>
+        </ul>
+
+      </section>
+
```

Comment:
- [ ] Please remove Author 2 from your readme since you are the only one working on this project, you will have time to add that later but for now, you are still working alone.